design/apidsl/action.go

```diff
@@ -415,10 +415,10 @@ func Payload(p interface{}, dsls ...func()) {
 //	OptionalPayload(BottlePayload)		// Request payload is described by the BottlePayload type and is optional
 //
 func OptionalPayload(p interface{}, dsls ...func()) {
-	payload(true, p, dsls...)
+	payload(true, false, p, dsls...)
 }
 
-func payload(isOptional bool, p interface{}, dsls ...func()) {
+func payload(isOptional, isMultipart bool, p interface{}, dsls ...func()) {
```

Comment:
See above, seems we don't need the `isMultipart` parameter now that we use a different DSL.

Reply:
Sorry, fixed it. https://github.com/goadesign/goa/pull/1582/commits/f7d765af7f76651b72c190e2c094f017f2ad25af